README.md

```diff
@@ -161,7 +161,18 @@ use this expiration which is set in mod-authtoken. The `RESET_PASSWORD_LINK_EXPI
 `RESET_PASSWORD_LINK_EXPIRATION_UNIT_OF_TIME` configuration parameters are used in the notification sent
 by the mod-users-bl module.
 
+`RESET_PASSWORD_LINK_EXPIRATION_UNIT_OF_TIME` defaults to `hours` and can be any of `minutes`, `hours`, `days`,
+and `weeks`.
```

Comment:
I think perhaps a sentence should be added here that this value needs to be the same as the expiration of the access token if refresh token rotation is being used since I believe the reset link contains a token and that token is an access token.

Reply:
Yes and I think this was actually added by me previously...